target/portfolio-1/index-style.css

```diff
@@ -0,0 +1,5 @@
+#map {
+  border: thin solid black;
```

Comment:
pull from dev so this file isn't here

Reply:
not entirely sure what you mean here, but pulled from dev on most recent push, hope that resolves this.